src/functional/iterator/embedded.py

```diff
@@ -129,6 +131,10 @@ def __array__(self) -> np.ndarray:
         ...
 
 
+def _is_column(v: Any) -> TypeGuard[Column]:
+    return isinstance(v, np.ndarray)
```

Comment:
mypy didn't like it, but then I thought it's actually more expressive to use _is_column because it allows for more than just isinstance if that make sense in the future.